CONTRIBUTING.md

```diff
@@ -111,32 +111,44 @@ $ ./gradlew integrationTest
 #### Running integration tests on Android
 
 
-Running the integration tests on android requires the running of a dedicated test app,
-in addition to the requirements for Java SE integration tests. 
+Running the integration tests on Android requires running the tests from an
+Android context. For this there is a Test App in the `AndroidTest` directory.
+The test app project will automatically pull in the source code from the
+sync-core and sync-android modules.
 
 The test application build scripts require a running emulator and the ```ANDROID_HOME```
  environment variable to be set
 
-The minimum requirements for an android emulator:
+The minimum requirements for an Android emulator;
 
 * Minimum API Level 15 (Target API Level is 20)
 * An SD card
 
-This test app can be run via gradle (in the AndroidTest directory of your checkout)
+The tests can be run like any other Android application test suite, through
+Android Studio / IntelliJ Idea or via the command line.
 
+You *must* run the `uploadFixtures` gradle task before running any tests.
+
+Note: you can run the `uploadFixtures` task once per emulator if you don't add any
+test fixtures.
+
+Steps to run the tests:
+
+1.  Start Android emulator
+2. Run the follow command from the root of the repo
 
 ```bash
-$ ../gradlew clean installStandardTestDebug waitForTestAppToFinishTests
+$ cd AndroidTest
+$ ../gradlew clean uploadFixtures connectedCheck
 ```
-The app will run all tests on specified by the build variant on first start up, to rerun tests
-you must rerun the gradle build.
+
 
 #### Collecting Test Results
 
-Test Results are displayed via the app, a test report is also available for CI builds.
-The test result report is in the same format as the ANT JUnit task reports. These style reports can be interpreted by 
-Jenkins and other systems.  The reports are located under the ``` build/test-results/ ``` the file name is automatically generated 
-by the build scripts, the base file name is ```testResults_``` it is suffixed with a UUID for that build.
+Test Results are collected in the usual manner for Android tests. If run via
```

Comment:
Link to a description of "the usual manner" for people like me who don't know what this is.